reports/performance/_outliers.qmd

```diff
@@ -368,3 +368,133 @@ qnt_df <- assessment_pin %>%
 qnt_df %>%
   datatable(rownames = FALSE)
 ```
+
+
+## Increase in Housing Value
+
+This section looks at the year over year change in assessed housing values. It separates the data into individual observations, identifying 10 PINs in each neighborhood which grew at the fastest rate as well as the median rate of change for each neighborhood. 
+
+::: panel-tabset
+
+## Map of Largest YOY Changes
+
+This map caps the legend at a 300% to increase legibility. It then plots the 10 largest year over year increases in assessed values by neighborhood. 
+
+```{r _outliers_map_of_largest_change}
+largest_increases <- assessment_pin %>%
+  group_by(meta_nbhd_code) %>%
+  left_join(ccao::nbhd_shp, by = c("meta_nbhd_code" = "town_nbhd")) %>%
+  filter(!is.na(prior_near_yoy_change_pct)) %>%
+  mutate(neighborhod_median_change = median(prior_near_yoy_change_pct * 100), ) %>%
+  arrange(desc(prior_near_yoy_change_pct)) %>%
+  slice(1:10) %>%
+  mutate(
+    prior_near_yoy_change_pct = prior_near_yoy_change_pct * 100,
+    capped_yoy_change = pmin(prior_near_yoy_change_pct, 300)
+  )
+
+
+nbhd_increases <- st_as_sf(largest_increases) %>%
+  distinct(meta_nbhd_code, .keep_all = TRUE)
+
+color_palette <-
+  colorNumeric(palette = c("white", "red"), domain = c(0, 300))
+
+leaflet() %>%
+  addProviderTiles(providers$CartoDB.Positron) %>%
+  addPolygons(
+    data = nbhd_increases,
+    fillColor = "white",
+    weight = 1,
+    color = "black",
+    opacity = 1,
+    fillOpacity = 0.5
+  ) %>%
+  addCircleMarkers(
+    data = largest_increases,
+    ~loc_longitude,
+    ~loc_latitude,
+    color = ~ color_palette(capped_yoy_change),
+    opacity = 0.7,
+    fillOpacity = 0.7,
+    radius = 1,
+    popup = ~ paste(
+      "Neighborhood:",
+      meta_nbhd_code,
+      "<br>Change:",
+      prior_near_yoy_change_pct,
+      "%"
+    )
+  ) %>%
+  addLegend(
+    "bottomright",
+    pal = color_palette,
+    values = c(0, 300),
+    title = "Year-over-Year Change (%)"
+  ) %>%
+  setView(
+    lng = mean(largest_increases$loc_longitude),
+    lat = mean(largest_increases$loc_latitude),
+    zoom = 10
+  )
+```
+
+## Table of Largest YOY Changes
+
+```{r _outliers_table_of_largest_change}
+datatable(
```

Comment:
**issue:** Please format the output columns nicely. Also, it's not useful to have > 700 pages of output to look at here. I would change this table to N largest tables by township rather than neighborhood, or find some other way to distill down this info.

Reply:
Seconded by township, also I think column *selection* needs a little refining here. I see things like prior far value but prior near percent change, etc.